### PR TITLE
first pass at new and updated usecases as presented at TPAC

### PR DESCRIPTION
**Pull request by steely-glint**

### CODE REVIEW

**Comment by aboba** [comment]
Can you present the PR at the December 2 virtual interim?  Slides are here:  https://docs.google.com/presentation/d/17Z_vR4pOXn-9wthiqbM9GC7JE7vfbdNffEedBPqj1Mw/

**Comment by steely-glint** [comment]
Yep, will do. 
I’ll add slides in the next couple of days

Thanks!
T.

File: index.html [hunk 1]
```diff
@@ -519,11 +563,102 @@ <h4>Untrusted JavaScript Cloud Conferencing</h4>
...
+         </tr>
+         <tr>
+            <td>NXX</td>
+            <td>Ability to reuse subtitle assets streamed over the data channels
+            </td>
+         </tr>
+       </tbody>
+   </table>  
+   <p>Experience: Both |pipe| and millicast have built systems of this sort.</p>
+</section>
```
Comment (agouaillard-cosmo) [inline index.html:639]: While millicast acknowledges that DRM support, Subtitle support, audio channel/track selection (multilanguage), subtitle track selection (multi language), are very frequent request in the field of broadcasting, we are not advocating for any specific implementation. Moreover, we do not support data channel at this stag, so we have not build a system based on the proposed implementation. We believe recent standards around rte payloads for timed text and other approaches might be more suitable to achieve the expected goal.

Comment (steely-glint) [reply index.html:639]: Thanks for the clarification. What I'm trying to indicate is that there is demand for this usecase, which we can't yet satisfy using the webRTC 1.0 spec. Both |pipe| and Millicast have (somewhat) solved these problems based on the current spec, albeit in a less than perfect way. 
Happy to drop the Millicast reference if you like.

**Comment by agouaillard-cosmo** [comment]
No need to drop the millicast reference if we keep the use case and do not speak about specific implementation . I.e drop “datachannel”

File: index.html [hunk 2]
```diff
@@ -519,11 +563,102 @@ <h4>Untrusted JavaScript Cloud Conferencing</h4>
...
+       <thead>
+          <tr>
+             <th>Requirement ID</th>
+             <th>Description</th>
+          </tr>
+       </thead>
+       <tbody>
+           <tr>
+              <td>NXX</td>
+              <td>A URI format that defines the remaining transport related fields eg service address/port, ICE credentials, DTLS fingerprint
```
Comment (henbos) [inline index.html:655]: This goes beyond specifying a use case, the URI is a proposed solution

**Pull request merged**